src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -355,6 +376,156 @@
+  </EnumProperty>
+
+  <!-- TODO: Shortlink for help URL -->
+  <EnumProperty Name="AuthenticationMode"
+                DisplayName="Authentication mode"
+                Description="Method for identifying the currently logged-on user within the application."
+                HelpUrl="https://docs.microsoft.com/dotnet/api/microsoft.visualbasic.applicationservices.authenticationmode"
+                Category="ApplicationFramework">
+    <EnumProperty.Metadata>
+      <NameValuePair Name="DependsOn" Value="Application::UseApplicationFramework" />
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>
+          (has-evaluated-value "Application" "UseApplicationFramework" true)
+        </NameValuePair.Value>
+      </NameValuePair>
+    </EnumProperty.Metadata>
+    <!-- TODO: Value provider logic needed to convert string enums to integer values -->
+    <!-- 0 - DisplayName="Windows authentication via My.User object" -->
+    <EnumValue Name="Windows"
+               DisplayName="Windows" />
+    <!-- 1 - DisplayName="Custom authentication implemented within the application" -->
+    <EnumValue Name="ApplicationDefined"
+               DisplayName="Application-defined" />
+  </EnumProperty>
+
+  <!-- TODO: Shortlink for help URL -->
+  <EnumProperty Name="ShutdownMode"
+                DisplayName="Shutdown mode"
+                Description="Indicates which condition causes the application to shut down."
+                HelpUrl="https://docs.microsoft.com/dotnet/api/microsoft.visualbasic.applicationservices.shutdownmode"
+                Category="ApplicationFramework">
+    <EnumProperty.Metadata>
+      <NameValuePair Name="DependsOn" Value="Application::UseApplicationFramework" />
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>
+          (has-evaluated-value "Application" "UseApplicationFramework" true)
+        </NameValuePair.Value>
+      </NameValuePair>
+    </EnumProperty.Metadata>
+    <!-- TODO: Value provider logic needed to convert string enums to integer values -->
+    <!-- 0 - DisplayName="Shut down when the main form closes" -->
+    <EnumValue Name="AfterMainFormCloses"
+               DisplayName="After main form closes" />
+    <!-- 1 - DisplayName="Shut down only after the last form closes" -->
+    <EnumValue Name="AfterAllFormsClose"
+               DisplayName="After last form closes" />
+  </EnumProperty>
+
+  <!-- TODO: Shortlink for help URL -->
+  <DynamicEnumProperty Name="SplashScreen"
+                       DisplayName="Splash screen"
+                       Description="Sets the form to be used as a splash screen for the application."
+                       HelpUrl="https://docs.microsoft.com/dotnet/api/microsoft.visualbasic.applicationservices.windowsformsapplicationbase.splashscreen"
+                       Category="ApplicationFramework"
+                       EnumProvider="StartupObjectsEnumProvider">
```

Comment:
This will need a custom provider to reflect on the forms available.

Reply:
@melytc is on the case with https://github.com/dotnet/project-system/pull/7936

Reply:
Splash Screen wasn't included in @KlausLoeffelmann spec, do we want to bring that in, too? I suspect it wouldn't be using the same EnumProvider as StartupObjects.

Reply:
Yes, we'd want splash screen. Based on what Klaus said at the start of the meeting (only having two days to write this up), I assumed he wouldn't catch everything. I'd need to do a similar investigation for the other pages (other than Application) to make sure we have property parity. **Additionally**, there are actually more properties that are related to this stuff than we expose, but I have to assume the target for now is parity and not additions.

Reply:
Good catch!
I absolutely missed that - yes, we need the SplashScreen.
I take a look tomorrow to see if there are any important things to look out for!